platform/pulpcore/app/views/status.py

```diff
@@ -0,0 +1,35 @@
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from pulpcore.app.serializers.task import WorkerSerializer
+import status
+
+
+class StatusView(APIView):
+    """
+    Returns status information about the application
+    """
+    authentication_classes = []
```

Comment:
I think it's worth a quick comment to explain why these are being set to empty lists.